packages/core/admin/server/validation/transfer/token.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const { yup, validateYupSchema } = require('@strapi/utils');
+const constants = require('../../services/constants');
+
+const transferTokenCreationSchema = yup
+  .object()
+  .shape({
+    name: yup.string().min(1).required(),
+    description: yup.string().optional(),
+    permissions: yup.array().of(yup.string()).nullable(),
+    lifespan: yup
+      .number()
+      .integer()
```

Comment:
```suggestion
      .number()
```
Given that we use `.oneOf` below, we might want to remove this to avoid bigint issues

Reply:
Indeed, should we update API token as well?

Reply:
yeah, as long as it stays in this feature branch long enough for it to get tested, I don't want to break something in an attempt to make it safer